kmmo/intel-dgpu-on-premise-build-mode.yaml

```diff
@@ -18,6 +18,7 @@ metadata:
 spec:
```

Comment:
Can we change the commit comments to 

kmmo: Set imagePullPolicy as Always in on-premise build mode

Set .spec.moduleLoader.container.imagePullPolicy to Always.
So KMM Operator can always pull the image from the registry
and use the new image built by on-premise mode 

For details about the image pull policy, please visit
https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting

Signed-off-by: Hersh Pathak hersh.pathak@intel.com

Reply:
Updated